dietza/Overlook
# Classes, Testing, ALL early build

### What is the change?
(too much)
The workflow to this point has been scattered, and this PR consequently affects all associated files.

### What does it fix?
Creates User and Manager classes,
Builds and passes unit tests for relevant properties and methods
Builds initial DOM displays and interactions

### Is this a fix or a feature?
Mostly features, building login and basic info dashboard displays for users
refactors / fixes along the way 

### Where should the reviewer start?
Line 1 in all changed files

### How should this be tested?
Run npm start in terminal and make sure all content for localHost: 8080 loads on the DOM and console is clear of bugs.
Run a Lighthouse audit in DevTools to check accessibility considerations.

### Any relevant tickets?
#3 , #6 , #7
